test/Spec.hs

```diff
@@ -0,0 +1,2 @@
+main :: IO ()
+main = putStrLn "Test suite not yet implemented"
```

Comment:
You can leverage this one as an excuse to get started with [QuickCheck](https://hackage.haskell.org/package/QuickCheck) - a leading property testing library.